mp/src/materialsystem/stdshaders/pbr_vs30.fxc

```diff
@@ -9,6 +9,7 @@
 //	DYNAMIC: "SKINNING"					"0..1"
 //  DYNAMIC: "LIGHTING_PREVIEW"			"0..1"
 //  DYNAMIC: "NUM_LIGHTS"				"0..4"
+//	STATIC:  "PARALLAXOCCLUSION"		"0..1"
```

Comment:
Do we need a separate toggle for self shadowing?

Reply:
i guess we need it, i havent put the self-shadowing code yet since performance penalty is still too high.

Reply:
the shader compiles for approx. 12 minutes now. (i5 4460, GTX970) so we might not want to add a `//static: PARALLAXSHADOW` should we implement parallax shadows because it will double the combo count for a few lines of code.

Reply:
Compilation time is less of an issue than the runtime of the shadows. You can add a SKIP to not enable shadows if parallax isn't enabled. That should only increase the combos by 50%.